src/scripts/main.js

```diff
@@ -1 +1,45 @@
 'use strict';
+
+function toggleScroll() {
+  const body = document.body;
+  const menuComputedStyle = window.getComputedStyle(menu);
+  const productComputedStyle = window.getComputedStyle(product);
+
+  if (menuComputedStyle.opacity === '1'
+  || productComputedStyle.opacity === '1') {
+    body.classList.add('no-scroll');
+  } else {
+    body.classList.remove('no-scroll');
+  }
+}
+
+const menu = document.getElementById('menu');
+const product = document.getElementById('product');
+
+function checkScreenWidth() {
+  const screenWidth = window.innerWidth
+  || document.documentElement.clientWidth;
+
+  if (screenWidth > 1280) {
+    menu.style.display = 'none';
```

Comment:
Directly manipulating the style of an element can lead to hard to track issues and lacks scalability. Consider using CSS classes and toggling them on or off using JavaScript.